architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,29 +175,143 @@ endif::[]
 [[image-streams]]
 == Image Streams
 
-An _image stream_ comprises any number of
```

Comment:
this whole file needs to be renamed, right?

Reply:
@bparees Yeah, I think that the Image Stream section should be its own page. I will see what the others think. You have any thoughts?

Reply:
yes they should be split out (also you seem to have updated the table of contents to imply there would only be an images page?  but i guess you'll want to add a builds entry to the table of contents).

Maybe do that in a follow up PR though, so we can keep this one to getting the content right?  There's been a lot of good discussion/review and i don't want to muddy the waters further.